### ISSUE TITLE
ItemGroup Remove is ignored (csproj file)

### ISSUE THREAD

**Issue by guillep2k** [fernandoescolar/vscode-solution-explorer#142]
Extension version: v0.3.10, VSCode 1.47.3

Hi! Thanks for this life saving extension. I'd like to report a mild annoyance about the parsing of `csproj` files, which is what I'm using.

In VS2019 I can exclude certain items from the browser (right-click on item, exclude from project). This creates an entry in the project file as follows:

#### For folders
```
  <ItemGroup>
    <Compile Remove="Folder-I-need-ignored\**" />
    <EmbeddedResource Remove="Folder-I-need-ignored\**" />
    <None Remove="Folder-I-need-ignored\**" />
  </ItemGroup>
```

#### For files
```
  <ItemGroup>
    <None Remove="File-I-need-ignored.txt" />
  </ItemGroup>
```

But in the solution explorer provided by the extension the `Remove` attribute seems to be ignored (i.e. files and folders are shown despite being excluded). This is expecially annoying for git submodules (which are loaded inside the project tree, but have their own entry in the solution file).

By the way, the `.git` directory is automatically ignored in Visual Studio; perhaps it should be ignored by the extension as well. It's showing up in the project tree.

**Comment by fernandoescolar** [comment]
Hi @guillep2k, 

Maybe the setting parameter bellow could help you: 

```
vssolution.netcoreIgnore
```

**Comment by guillep2k** [comment]
> 
> 
> Hi @guillep2k,
> 
> Maybe the setting parameter bellow could help you:
> 
> ```
> vssolution.netcoreIgnore
> ```

Yes, it does help, but in our projects some people work with VS2019 and some with VSCode. It would be great if VSCode would pick up the project ignores instead of maintaining a separate configuration.

**Comment by ygrinevich** [comment]
Can you please add support for "*.extension" wildcard remove
